### DISCUSSION TITLE
Is is possible to have multiple Operation Object for single callback event URL and can a single callback event have multiple URLs ? Does it make sense ?

### DISCUSSION THREAD

**Discussion by mma5997** [OAI/OpenAPI-Specification#2579] [Q&A]
Case1:
Can a single callback event have multiple operations - ex POST, PUT, PATCH etc...

![image](https://user-images.githubusercontent.com/34859630/118691459-baf7c480-b826-11eb-9798-519b6c63f7cb.png)

Case2:
Can a single callback event have multiple URLs as denoted below?

![image](https://user-images.githubusercontent.com/34859630/118691252-8421ae80-b826-11eb-8733-c34aa120b94f.png)

My understanding is that an event should be causing callback request invocation, 
and usually a request is configured with a particular URL and a particular operation.
So looks one to one mapping between 
Event ==> URL and
URL ==> Operation

But is it possible

one to many for both below
Event ==> URL and
URL ==> Operation

Thanks!

**Comment by handrews** [comment]
The keys under `MyEvent` are technically runtime expressions that produce URLs, rather than literal URLs, but yes you can have multiple of them.  The values are Path Item Objects, so yes, they can have multiple operations as well.

Since a Callback Object defines _possible_ callbacks, it's plausible that there might be possible callbacks to several different URLs with several different HTTP methods.  Whether that "makes sense" will depend entirely on your application.

Since this is an old question (the project kind of went into hibernation during much of the pandemic), and the answer is essentially "yes, you can do that", I'm going to close it as answered.  If I missed something, please comment and I can re-open it.
